src/Storage/StorageInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Storage;
+
+use Yiisoft\Yii\Debug\Collector\CollectorInterface;
+
+/**
+ * Debug data storage responsibility is to store debug data from collectors added
+ */
+interface StorageInterface
+{
+    /**
+     * Add collector to get debug data from
+     * @param CollectorInterface $collector
+     */
+    public function addCollector(CollectorInterface $collector): void;
+
+    public function getData(): array;
```

Comment:
I don't think this method has to be part of the interface. It is implementation detail nothing but tests use.